src/main/java/com/coolpeace/domain/accommodation/service/AccomodationService.java

```diff
@@ -0,0 +1,53 @@
+package com.coolpeace.domain.accommodation.service;
+
+import com.coolpeace.domain.accommodation.dto.response.AccomodationResponse;
+import com.coolpeace.domain.accommodation.dto.response.RoomResponse;
+import com.coolpeace.domain.accommodation.entity.Accommodation;
+import com.coolpeace.domain.accommodation.exception.AccommodationNotFoundException;
+import com.coolpeace.domain.accommodation.repository.AccommodationRepository;
+import com.coolpeace.domain.room.entity.Room;
+import com.coolpeace.domain.room.repository.RoomRepository;
+import com.coolpeace.global.jwt.security.JwtPrincipal;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class AccomodationService {
+
+    private final AccommodationRepository accommodationRepository;
+    private final RoomRepository roomRepository;
+
+    public List<AccomodationResponse> getAccommodations(JwtPrincipal jwtPrincipal) {
+
+        Integer memberId = Integer.parseInt(jwtPrincipal.getMemberId());
+
+        List<Accommodation> accommodations
+            = accommodationRepository.findAllByMemberId(memberId);
+
+        List<AccomodationResponse> accomodationResponses = new ArrayList<>();
+        for (Accommodation accommodation : accommodations) {
+            accomodationResponses.add(AccomodationResponse.fromEntity(accommodation));
+        }
+
+        return accomodationResponses;
```

Comment:
```suggestion
Integer memberId = Integer.parseInt(jwtPrincipal.getMemberId());

return accommodationRepository.findAllByMemberId(memberId)
                    .stream().map(AccomodationResponse::fromEntity).toList();
```

이런식으로 스트림을 써보는건 어떨까요? 좀더 가독성이 있어보일 수 있을 것 같습니다 😄

Reply:
네 스트림이 더 좋을거 같아요